### ISSUE TITLE
Removed processMarkdown in Adaptive Cards 1.1

### ISSUE THREAD

**Issue by Tosters** [Microsoft/BotFramework-WebChat#1510]
Hi,
In the following line from the [link](https://github.com/Microsoft/BotFramework-WebChat/blob/d35a7acd175148ef26fae9ff0813efc843962dca/packages/component/src/Attachment/AdaptiveCardRenderer.js#L92) 'processMarkdown' property is being set, but in 'adaptivecards' npm package 'card-elements.js' file it is said that that event has been removed ([link](https://github.com/Microsoft/AdaptiveCards/blob/c4d515e828e340557d2c448aec32da5576750ec5/source/nodejs/adaptivecards/src/card-elements.ts#L5457) in line 5457).
Is this is a bug or there is some special way to fix this? Fixing the package can resolve this, but this will break the updates later on.

**Comment by compulim** [comment]
Looks like they should deprecate it peacefully, but they just throw an error.

The current version of Web Chat is using Adaptive Cards 1.0 and it is still fine. When we bump to AC 1.1, we will sure need to fix this.

We have some plans on versioning:

- Web Chat latest will match AC latest (with a few weeks of lag)
- We will document ways to use Web Chat with different version of AC
- There are some works to decouple AC from Web Chat
   - #1302 and #1304 is to move AC from our `botframework-webchat-component` to `botframework-webchat` package
   - #1259 is to move rich card to HTML renderer, for better design/layout

If you want to move to AC 1.1, it's doable, supported, not hacky, but it is not trivial (because we have too much coupling).

- Create your own set of [`AdaptiveCardAttachment.js`](https://github.com/Microsoft/BotFramework-WebChat/blob/master/packages/component/src/Attachment/AdaptiveCardAttachment.js) and [`AdaptiveCardRenderer.js`](https://github.com/Microsoft/BotFramework-WebChat/blob/master/packages/component/src/Attachment/AdaptiveCardRenderer.js) that use the newer AC 1.1 package
- Using attachment middleware to register a new set of middleware that will process Adaptive Card content
   - You can copy from [`createAdaptiveCardMiddleware.js`](https://github.com/Microsoft/BotFramework-WebChat/blob/master/packages/component/src/Middleware/Attachment/createAdaptiveCardMiddleware.js) and remove most chunk of it, just left this branch `attachment.contentType === 'application/vnd.microsoft.card.adaptive'`

**Comment by Tosters** [comment]
Hi,
Thank you for the info.

Just ran the `npx create-react-app my-app` command to create new react app and after that `npm i botframework-webchat` to install the web chat.
This, as you mentioned, produced dependencies in botframework-webchat-component
"adaptivecards": "^1.0.0"
but version that is installed in projects node_module folder is "1.1.1"

**Comment by elena20ruiz** [comment]
I had the same problem with React App, and the reason is what @Tosters explained very well. 
On `botframework-webchat` the dependency  is `"adaptivecards": "^1.0.0"`.

Finally, I have included the dependence `'adaptivecards'` in my project in order to use the version `1.0.0`.
So, I have done this changes on my `package.json`, and now is working:
**Before:** ( I didn't have adapitvecard dependency)
```
[...]
"botframework-webchat": "^4.1.0",
[...]
```
**Now:**
```
[...]
"botframework-webchat": "4.1.0",
"adaptivecards": "1.0.0",
[...]
```

I now is a temporal solution, but this problem was very unexpected.

**Comment by Tosters** [comment]
Yep, simply installed @1.0.0. This is a better way than editing the package itself.
Thanks.

**Comment by compulim** [comment]
I didn't figure out it was Adaptive Cards not following semver, we are using `@^1.0.0`.

Will lock it down to `adaptivecards@1.0.0` so it won't break more people.

**Comment by compulim** [comment]
@corinagum can you help on this one?

**Comment by compulim** [comment]
PR #1558 is merged and it bumped to `adaptivecards@~1.1.2`. It will lock down the major and minor version.
